runtime/vm/MHInterpreter.hpp

```diff
@@ -396,6 +396,9 @@ class VM_MHInterpreter
 	{
 		switch (returnType) {
 		case J9NtcBoolean:
+#if defined(OSX) && defined(J9AARCH64)
```

Comment:
Same comment as above on documenting why this specialization of behaviour is necessary.

Reply:
Same here.  I stopped adding the `#if defined(OSX) && defined(J9AARCH64)` code for the return value part.